org-cyf-itp/content/data-groups/success/index.md

```diff
@@ -1,15 +1,36 @@
 +++
 title = 'success'
-description = 'How do we know if we have completed this module?'
+description = 'How do we know we have completed User Data Groups?'
 layout = 'success'
 emoji= '✅'
 menu_level = ['module']
 weight = 11
 [[objectives]]
-1="Use data structures to model some part of a problem domain"
-2="Identify acceptance criteria for a problem"
-3="Break a problem down into different test cases"
-4="Build tests to constrain the output of an LLM"
-5="Solve at least 6 6kyu problems in Codewars"
-6="Solve up to level 20 in the Bandit repeatedly"
+1="Link to your PR for [Sprint 1 Coursework](https://github.com/CodeYourFuture/Module-Data-Groups/issues/15)"
+2="Link to your PR for [Sprint 2 Coursework](https://github.com/CodeYourFuture/Module-Data-Groups/issues/14)"
+3="Link to your PR for your [Alarm Clock App](https://github.com/CodeYourFuture/Module-Data-Groups/issues/26)"
+4="Link to your PR for your [Quote Generator App](https://github.com/CodeYourFuture/Module-Data-Groups/issues/20)"
+5="Link to your PR to your [Reading List Display](https://github.com/CodeYourFuture/Module-Data-Groups/issues/18)"
```

Comment:
I'm wondering if we need links all of these apps (alarm clock, quote generator, reading list) 🤔 
If we feel like there's some duplication in what skills are needed to do these - maybe we remove one?